entry_types/scrolled/package/spec/contentElements/dataWrapperChart/useIframeHeight-spec.js

```diff
@@ -0,0 +1,19 @@
+import {useIframeHeight} from '../../../src/contentElements/dataWrapperChart/useIframeHeight'
+
+import {renderHook} from '@testing-library/react-hooks';
+import {fakeParentWindow, tick} from 'support';
+import {act} from '@testing-library/react';
+
+describe('useIframeHeight', () => {
+  it('sets the default height', async () => {
+    const testURL = 'https://datawrapper.dwcdn.net/CXXQo/1/';
```

Comment:
While it is nice to have a sample URL here to remind us of the expected format, it might be safer to use a mock instead of a real graph. Maybe something like `https://datawrapper.dwcdn.net/aBcDe/1/`?

This would also protect us against hypothetical side effects that occur just because we provide the URL of a specific graph, i. e. a spec that passes only by accident